LocalPackages/Freemium/Sources/Freemium/FreemiumPIRState.swift

```diff
@@ -17,16 +17,19 @@
 //
 
 import Foundation
+import Subscription
 
 /// `FreemiumPIRState` types provide access to Freemium PIR-related state
-protocol FreemiumPIRState {
+public protocol FreemiumPIRState {
     var didOnboard: Bool { get set }
+    var isCurrentUser: Bool { get }
 }
 
 /// Default implementation of `FreemiumPIRState`. `UserDefaults` is used as underlying storage.
 public final class DefaultFreemiumPIRState: FreemiumPIRState {
 
     private let userDefaults: UserDefaults
+    private let accountManager: AccountManager
     private let key = "macos.browser.freemium.pir"
```

Comment:
This probably should be specific key for `didOnboard` property

Reply:
Agree, missed this.